dissect/target/plugins/os/windows/lnk.py

```diff
@@ -12,20 +13,20 @@
 LnkRecord = TargetRecordDescriptor(
     "windows/filesystem/lnk",
     [
-        ("uri", "lnk_path"),
+        ("path", "lnk_path"),
         ("string", "lnk_name"),
         ("datetime", "lnk_mtime"),
         ("datetime", "lnk_atime"),
         ("datetime", "lnk_ctime"),
         ("string", "lnk_relativepath"),
```

Comment:
The field `lnk_relativepath` has the field type `string` but the following values are parsed when using the plugin?

* `..\..\..\..\..\Program Files\Wireshark\Wireshark.exe`
* `..\..\..\..\..\Program Files\Microsoft Office\root\Office16\MSACCESS.EXE`
* `.\pestudio.exe`

Can the field type `path` be used for this field? Or is the usage of the fieldtype `string` best practice because the values are not full Windows paths like `C:\Program Files\Wireshark\Wireshark.exe`.

Reply:
Okay